test/integration/build_to_git_test.go

```diff
@@ -0,0 +1,170 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package integration_test
```

Comment:
Shouldn't this just be "package integration"?

Reply:
Keep use `integration_test` package name.